pkg/apis/build/v1alpha2/image_builds.go

```diff
@@ -58,6 +58,7 @@ func (im *Image) Build(sourceResolver *SourceResolver, builder BuilderResource,
 		Spec: BuildSpec{
 			Tags:                  im.generateTags(buildNumber),
 			Builder:               builder.BuildBuilderSpec(),
+			RunImage:              builder.RunImage(),
```

Comment:
Based on [comment](https://github.com/pivotal/kpack/pull/988/files#r918365344):

```golang
RunImage: BuildSpecImage{
    Image: builder.RunImage(),
},
```

Reply:
Ya this is exactly what I had in mind. Maybe we could reuse the spec from the cluster stack and move it to core api. No strong preference though.